### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by scottvaughanamido**

### CODE REVIEW

File: .template.config/template.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "WebAPI",
     "CQRS"
   ],
-  "name": "Amido Stacks Web Api CQRS - Full solution",
+  "name": "Amido Stacks Web Api CQRS",
```
Comment (underscoreHao) [inline .template.config/template.json:9]: It should be `Web API` since that's the way Microsoft list their api templates

File: README.md [hunk 2]
```diff
@@ -11,56 +17,21 @@ All templates from this repository come as part of the [Amido.Stacks.CQRS.Templa
 
 #### Template installation
 
-To install the templates on your own machine you'll need to download the [Amido.Stacks.CQRS.Templates](https://www.nuget.org/packages/Amido.Stacks.CQRS.Templates/) NuGet package.
-
-Then you can install it locally via the command line (command is given with an example version)
+For the latest template version, please consult the Nuget page [Amido.Stacks.CQRS.Events.Templates](https://www.nuget.org/packages/Amido.Stacks.CQRS.Templates/). To install the templates to your machine via the command line:
```
Comment (underscoreHao) [inline README.md:20]: Name should be `Amido.Stacks.CQRS`. The link is correct.

File: .template.config/template.json (hunk 1 above)
Comment (scottvaughanamido) [reply .template.config/template.json]: Changed to "Amido Stacks Web API CQRS"

File: README.md (hunk 2 above)
Comment (scottvaughanamido) [reply README.md]: Link changed to Amido.Stacks.CQRS

File: README.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ The output will list all installed templates (not listed for brevity). In that l
 Template Name                                         Short Name                       Language    Tags
 ----------------------------------------------------  -------------------------------  ----------  ------------------------------------------
 ...
-Amido Stacks Web Api CQRS - Add to existing solution  stacks-add-web-api-cqrs          [C#]        Stacks/WebAPI/CQRS/api
+Amido Stacks Web API                                  stacks-add-web-api-cqrs          [C#]        Stacks/WebAPI/CQRS/api
```
Comment (underscoreHao) [inline README.md:32]: Isn't this `Amido Stacks CQRS Projects` now?

File: README.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ The output will list all installed templates (not listed for brevity). In that l
...
 Amido Stacks Web Api CQRS - Full solution             stacks-app-web-api-cqrs          [C#]        Stacks/WebAPI/CQRS/api
```
Comment (underscoreHao) [inline README.md:33]: And this should be the `Amido Stacks CQRS Web API`

File: README.md (hunk 3 above)
Comment (scottvaughanamido) [reply README.md]: Changed name of console output to reflect template name

File: README.md (hunk 4 above)
Comment (scottvaughanamido) [reply README.md]: Changed name of console output to reflect template name

**Review by underscoreHao — APPROVED** [review]

**Pull request merged**